docs/crds/gadgets/filetop.md

```diff
@@ -8,7 +8,7 @@ filetop shows reads and writes by file, with container details.
 The following parameters are supported:
  - interval: Output interval, in seconds. (default 1)
  - max_rows: Maximum rows to print. (default 20)
- - sort_by: The field to sort the results by (node,namespace,pod,container,mntns,pid,tid,comm,reads,writes,rbytes,wbytes,T,file). (default -reads,-writes,-rbytes,-wbytes)
+ - sort_by: The field to sort the results by (node,namespace,pod,container,hostnetwork,mntns,pid,tid,comm,reads,writes,rbytes,wbytes,T,file). (default -reads,-writes,-rbytes,-wbytes)
```

Comment:
It's now called `sort` and the help is something like: `--sort string            Sort by columns. Join multiple columns with ','. Prefix a column with '-' to sort in descending order. (default "-reads,-writes,-rbytes,-wbytes")`

IMO we can remove them from the documentation. It's too difficult to keep that updated.

Reply:
This is the CRD documentation. The text is automatically generated with `make generate-documentation`.